### DISCUSSION TITLE
Institution email verification

### DISCUSSION THREAD

**Discussion by Balogunjamiu** [community/community#65574] [GitHub Education]
### Hi everyone,

My institute is not responding to my mails please what other way can I use to get this educational offer

**Comment by fsantamaria1** [comment]
In my case, I uploaded a picture of an email I got from my professor congratulating me for passing my class. I think that as long as they can see that it was sent to your .edu email and is not a really old email, they will approve it. The email I uploaded was 4 months old, so I don't think it has to be extremely recent either.
